Dockerfile.bak.bak

```
@@ -0,0 +1,90 @@
+# Install Python packages (including mysqlclient)
+COPY requirements/base.txt requirements/dev.txt /tmp/
+RUN --mount=type=cache,target=/root/.cache/pip \
+    pip install --prefix=/install --no-cache-dir -r /tmp/base.txt && \
+    pip install --prefix=/install --no-cache-dir -r /tmp/dev.txt && \
+    pip install --prefix=/install mysqlclient==2.2.7 --global-option=build_ext \
+    --global-option="-I/usr/include/mysql" \
+    --global-option="-L/usr/lib/x86_64-linux-gnu" \
+    --global-option="-lmysqlclient"
+
+# Copy Doppler CLI install script
+RUN curl -sLf --retry 3 --tlsv1.2 --proto "=https" https://packages.doppler.com/public/cli/install.sh | sh -s -- --no-install-cli-version-file --install-path /install/bin
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)